react/week4/githubsearch/src/components/UserDetails.js

```diff
@@ -0,0 +1,36 @@
+import React, { useState, useEffect } from "react";
+
+function UserDetails({ user }) {
+  const [details, setDetails] = useState("");
```

Comment:
It's good practice to have our initial state the same data type. If our variable will hold an array, then we initialize it with `[]`, if it's a string, then `""`, if an object then `{}`, etc. So when we look at this line, we know `details` is an `object` and not a `string`. Also, to keep in mind, when working with input fields, and resetting the state to a different data type, you will get some weird errors.
```suggestion
  const [details, setDetails] = useState({});
```